openshift-knative-operator/cmd/operator/kodata/knative-serving/0.17.3/4-serving-post-install-jobs.yaml

```diff
@@ -18,7 +18,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: storage-version-migration-serving-0.17.3
+  generateName: storage-version-migration-serving-
```

Comment:
if we remove the `-serving` from the `generatedName`, I think we have just one `serving`, right?

Reply:
Yeah, but this comes via upstream. We could mangle it, but I opted to drop all upstream manifest mangling for now.